py/src/ai/h2o/sparkling/SharedBackendConf.py

```diff
@@ -314,6 +314,7 @@ def ignore_spark_public_dns(self):
         return self.ignoreSparkPublicDNS()
 
     def ignoreSparkPublicDNS(self):
+        warnings.warn("Method 'ignoreSparkPublicDNS' is deprecated in release 3.30 without replacement.")
```

Comment:
`"Method 'ignoreSparkPublicDNS' is deprecated in release 3.30 without replacement."` -> `"Method 'ignoreSparkPublicDNS' is deprecated. It will be removed in the release 3.30 without replacement."`?

Reply:
👍